windows-release/checkout.yml

```
@@ -19,3 +19,11 @@ steps:
 - script: git clone --progress -v --depth ${{ parameters.depth }} --branch $(Build.SourceBranchName) --single-branch $(Build.Repository.Uri) .
   displayName: 'git clone'
   condition: and(succeeded(), and(not(variables['GitRemote']), not(variables['SourceTag'])))
+
+- powershell: |
+    $checkout_commit = (git rev-parse HEAD)
+    if ($checkout_commit -ne SourceCommit) {
```

```suggestion
    if ($checkout_commit -ne '$(SourceCommit)') {
```

This should be fine, though injecting pipeline variables into Powershell can be a bit flakey...